client/src/containers/Workspaces/Workspaces.js

```diff
@@ -1,69 +1,58 @@
-import React, { useEffect, useState } from 'react';
-import { Link } from 'react-router-dom';
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
 
-import Container from '../../components/Container/Container';
-import Columns from '../../components/Columns/Columns';
-import InputField from '../../components/InputField/InputField';
-import SidebarList from '../../components/SidebarList/SidebarList';
-import Spinner  from '../../components/Spinner/Spinner';
- 
+import Container from "../../components/Container/Container";
+import Columns from "../../components/Columns/Columns";
+import InputField from "../../components/InputField/InputField";
+import SidebarList from "../../components/SidebarList/SidebarList";
+import Spinner from "../../components/Spinner/Spinner";
+import useFetch from "../../hooks/useFetch";
 
 const Workspaces = () => {
-  const [ownedWS, setOwnedWS] = useState([]);
-  const [ joinedWS, setJoinedWS] = useState([]);
-  const [ isLoading, setLoading ] = useState(true);
-
-  const ownedWorkspaces = ownedWS.map(workspace => (
-    <Link key={workspace._id} to={`/workspaces/${workspace._id}`}>{workspace.name}</Link>
-  ))
-
-  const joinedWorkspaces = joinedWS.map(workspace => (
-    <Link key={workspace._id} to={`/workspaces/${workspace._id}`}>{workspace.name}</Link>
-  ))
-
-  useEffect(() => {
-    // TODO: fetch workspaces list
-    const dummyWorkspaces = [
-      {
-        _id: 1,
-        name: 'Workspace 1'
-      },
-      {
-        _id: 2,
-        name: 'Workspace 2'
-      }
-    ]
-    setOwnedWS(dummyWorkspaces);
-    setJoinedWS(dummyWorkspaces);
-    setLoading(false);
-  }, []);
+  const { response, error, isLoading } = useFetch("GET", "/workspace", {});
 
   const dummyUser = {
     _id: 101010,
-    name: 'John',
-  }
+    name: "John"
```

Comment:
Do we need dummy user here? We have backend implemented. You use real data.

Reply:
Reverted this file